todolist copy 3/main.js

```diff
@@ -0,0 +1,74 @@
+const addTaskButton = document.getElementById('addTask');
+const deskTaskInput = document.getElementById('taskInput');
+const toDoList = document.getElementById('tasklist');
+const clearing = document.getElementById('clearAll');
+
+
+let tasks = [];

```

Comment:
we can rewrite it with nullish coalescing operator `let tasks = JSON.parse(localStorage.getItem("tasks")) ?? [];`
https://javascript.info/nullish-coalescing-operator